### ISSUE TITLE
Browse - directory modified date column misalignment

### ISSUE THREAD

**Issue by jungle-boogie** [mholt/caddy#1952]
### 1. What version of Caddy are you using?

Caddy 0.10.10 (+9a22cda Sun Nov 12 22:13:44 UTC 2017) (unofficial)

### 2. What are you trying to do?

Use the `browse` feature without the Modified column being misaligned 

### 3. What is your entire Caddyfile?
```text
browse
```

### 4. How did you run Caddy (give the full command and describe the execution environment)?

`sudo caddy -conf /usr/local/www/Caddyfile`

### 6. What did you expect to see?

Properly aligned columns. All columns but the modified are correct.

### 7. What did you see instead (give full error messages and/or log)?

[Imgur](https://i.imgur.com/VduuKvz.png) <-- pictorial description of the problem. Notice the Modified column isn't correct. 

### 8. How can someone who is starting from scratch reproduce the bug as minimally as possible?

Probably just using the `browse` feature, you will notice the alignment is off in the modified column.

On a related note, how can the YYYY-MM-DD format be implemented within this `browse` feature?

**Comment by jungle-boogie** [comment]
On further review, I think this would be corrected if all DD and MM used two digits.
The clock should also be 24hours.

Before:
```
6/6/2017, 4:24:17 PM
7/10/2017, 12:09:05 AM
```

After:
```
06/06/2017, 14:24:17
07/10/2017, 12:09:05
```

I'd still prefer a YYYY-MM-DD 24hr. 

Or something like this:
`11-Nov-2017 01:04`
From here:
https://ftp.openbsd.org/pub/OpenBSD/snapshots/octeon/

**Comment by tobya** [comment]
I think you can specify your own template for the browse directive, which would allow you to specify how you wanted the date to be formatted.

**Comment by jungle-boogie** [comment]
Thanks for the input @tobya.

I'd still like to see ISO 8601 implemented in the default template.

**Comment by tobya** [comment]
We are always delighted to accept a pull request for an update.  I will mark this as a feature request.

**Comment by jungle-boogie** [comment]
Hi @tobya, there's a pull request to align the column: https://github.com/mholt/caddy/pull/1954

**Comment by tobya** [comment]
This was fixed in #1954
